ui/src/app/views/app/component.html

```diff
@@ -1,26 +1,41 @@
-<div style="display: flex; justify-content: space-between;">
-  <div>
-    <img class="main-logo" src="assets/images/all-of-us-logo.svg" alt="All of Us"
-      style="width: 18rem;">
-    <img class="portal-logo" src="assets/images/portal-logo.svg" alt="Researcher Portal"
-      style="margin-left: 0.5rem; width: 13rem;">
+<ng-template *ngIf="(user | async).isSignedIn; then signedIn else notSignedIn"></ng-template>
+<ng-template #logoImages>
+  <img class="main-logo" src="assets/images/all-of-us-logo.svg" alt="All of Us"
+    style="height: 100%;">
+  <img class="portal-logo" src="assets/images/portal-logo.svg" alt="Researcher Portal"
+    style="margin-left: 0.5rem; height: 100%;">
+</ng-template>
+<ng-template #notSignedIn>
+  <div class="sign-in-container">
+    <div class="sign-in-inner-container">
+      <div style="height: 2rem; margin: 0.5rem 2rem;">
+        <ng-template *ngIf="true; then logoImages"></ng-template>
```

Comment:
What is this even accomplishing?? It looks like you have an if statement that always evaluates true, and you have the same statement repeated in two branches of the signed in versus not sign in decision. How is this different than a field that is always present?